src/Application/Commands/CallFrequentTasks.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Application\Commands;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'matchbot:tick',
+    description: 'Calls all per-minute commands; currently to expire old matching and send statistics'
+)]
+class CallFrequentTasks extends LockingCommand
```

Comment:
liking having a CallFrequentTasks command a lot if it means we can add new ones without having to make infrastructure repo changes. We could remove some of the duplication here but not much point while its just calling two commands. Eventually I still think using something like https://github.com/peppeocchi/php-cron-scheduler would make sense.

Reply:
Replied on slack